roles/rabbitmq/tasks/main.yml

```diff
@@ -15,3 +15,13 @@
     enabled: yes
     state: started
   tags: rabbitmq
+
+- name: Add RabbitMQ plugins
+  become: yes
+  rabbitmq_plugin:
+    name: "{{ item }}"
+    state: enabled
+    prefix: "{{ is_redhat6 | ternary(rmq_redhat6_prefix, omit) }}"
```

Comment:
Any comment/explanation would be nice to understand why `prefix` is needed on el6-based systems

Reply:
The rabbitmq RPM in EL 6 systems only includes the rabbitmq-plugins binary at `/usr/lib/rabbitmq/bin/rabbitmq-plugins` (see the list of files in https://centos.pkgs.org/6/epel-x86_64/rabbitmq-server-3.1.5-1.el6.noarch.rpm.html). In the EL 7 RPM, the rabbitmq-plugins binary is also available via `/usr/sbin/rabbitmq-plugins` (see the list of files in https://centos.pkgs.org/7/epel-x86_64/rabbitmq-server-3.3.5-34.el7.noarch.rpm.html). As the binary is already on the path in EL7, but not in EL6, we have to use a prefix to add `/usr/lib/rabbitmq/bin/` to the search path in EL6.

I will try write something more concise to leave a comment in the playbook.